src/riak_ql_parser.yrl

```diff
@@ -167,8 +169,8 @@ ColumnDefinition ->
 ColumnConstraint -> not_null : '$1'.
 
 DataType -> datetime : '$1'.
-DataType -> float : '$1'.
-DataType -> int : '$1'.
+DataType -> float_type : '$1'.
```

Comment:
The DDL compiler will choke on a record that looks like this

#riak_field_v1{
		name = "temperature_k",
	    position = 2,
		type = int_type,
 		optional = true}